app/src/main/java/com/vitorpamplona/amethyst/ui/navigation/DrawerContent.kt

```diff
@@ -254,7 +254,7 @@ fun ListContent(
         Spacer(modifier = Modifier.weight(1f))
 
         IconRow(
-            "Logout",
+            stringResource(R.string.log_out),
```

Comment:
Since #182 was already merged before I could fix this and it's just a 1-liner, I am dropping it here for simplicity.